dataclients/kubernetes/clusterstate.go

```diff
@@ -64,11 +64,15 @@ func (state *clusterState) getEndpointsByService(namespace, name, protocol strin
 	return targets
 }
 
+func (state *clusterState) GetEndpointsByTarget(namespace, name, protocol string, target *definitions.BackendPort) []string {
```

Comment:
I think `getEndpointsByTarget` also could be just made public

Reply:
done